charts/karmada-operator/README.md

```diff
@@ -0,0 +1,71 @@
+# karmada-operator
+
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: latest](https://img.shields.io/badge/AppVersion-latest-informational?style=flat-square)
+
+The Karmada operator is a method for installing, upgrading, and deleting Karmada instances. It builds upon the basic Karmada resource and controller concepts, provides convenience to centrally manage entire lifecyle of Karmada instances in a global cluster. With the operator, you can extend Karmada with custom resources (CRs) to manage your instances not only in local
+clusters but also in remote clusters.
+
+## TL;DR
+
+Switch to the `root` directory of the repo.
+
+```console
+helm install karmada-operator -n karmada-system --create-namespace --dependency-update ./charts/karmada-operator
+```
+
+## Prerequisites
+
+- Kubernetes 1.16+
+- helm v3+
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://charts.bitnami.com/bitnami | common | 1.x.x |
+
+## Installing the Chart
+
+Go to the root directory of the `karmada-io/karmada` repo. To install the Helm Chart with
+the release name `karmada-operator` in the namespace `karmada-system`, simply run the helm command:
+
+```shell
+helm install karmada-operator -n karmada-system  --create-namespace ./charts/karmada-operator --debug
+```
+
+```console
+kubectl get po -n karmada-system
+NAME                               READY   STATUS    RESTARTS   AGE
+karmada-operator-df95586f6-d5bll   1/1     Running   0          6s
+```
+
+## Uninstalling the Chart
```

Comment:
Seems some content is missing between install and uninstall. You know, the purpose of installation is not to uninstall 🥲 .

What's the next after installing charts? only uninstall?

Reply:
upgrade? I add upgrading content with installing together